pages/pil/read/views/index.jsx

```diff
@@ -1,31 +1,42 @@
 import React, { Fragment } from 'react';
 import { connect } from 'react-redux';
+import dateFormatter from 'date-fns/format';
+import { dateFormat } from '../../../../constants';
 import {
+  Link,
   Snippet,
   ModelSummary,
   Header
 } from '@asl/components';
 
-const formatters = url => ({
-  status: {
-    format: status => {
-      if (status !== 'active') {
-        return <Snippet url={`${url}/edit`}>status.inactive</Snippet>;
-      }
-    }
+const formatters = {
+  issueDate: {
+    format: issueDate => dateFormatter(issueDate, dateFormat.medium)
+  },
+  revocationDate: {
+    format: revocationDate => dateFormatter(revocationDate, dateFormat.medium)
   }
-});
+};
+
+const PIL = ({ model, profile }) => {
+  return (
+    <Fragment>
+      <Header
+        title={<Snippet>title</Snippet>}
+        subtitle={profile.name}
+      />
+      { model.status === 'active'
+        ? <ModelSummary model={model} formatters={formatters} />
+        : <div><Link page="pil.update" pilId={model.id} label={<Snippet>action.applyNow</Snippet>} className="govuk-button" /></div>
```

Comment:
What's the `<div>` doing here?

Reply:
It's to prevent the two links showing inline.